### DISCUSSION TITLE
Possible BUG with OSM&Maps and setting Auto Zoom map to "No Auto Zoom"

### DISCUSSION THREAD

**Discussion by Rmacca** [osmandapp/OsmAnd#20431] [Q&A]
Guys... this follows on from previous post (https://github.com/osmandapp/OsmAnd/discussions/19895)

**[quote]**
_I have been using the “OSMandMaps (Pro) version” App for a few years now and I have recently started to notice that I am having some issues with the App on my iPad and to some extent on my iPhone as well. I am running the latest IOS software on both ios devices. My 10’’ iPad is the 7th generation (built-in GPS receiver) iPad, and my iPhone is the iPhone-13.
The issues I am having are as follows:

"At random times, I have had the OSMand Map App just shut down for no reason during my trips while I’ve been using the App to navigate. I then must restart the App again to ensure that the recording of my current TRIP Tracklog continues, otherwise I have found that it does not record my current Tracklog. This has happened on both my Apple iPad and the iPhone 13 devices."_
**[/quote]**

On my last remote touring trip I had the same issue as mentioned above start to happen to me again. I then started to go through what settings I have changed since this issue started happening and I found something very interesting...

When I had the following settings, this is when I was having the crashing issues: 
From within "Settings" > "Driving" Base Profile >> Navigational Settings >> Map during Navigation >> Auto-centre map view >> 20 seconds
From within "Settings" > "Driving" Base Profile >> Navigational Settings >> Map during Navigation >> Auto Zoom Map >> No Auto Zoom
From within "Settings" > "Driving" Base Profile >> Navigational Settings >> Map during Navigation >> Snap to road >> "ON"

I then reset the settings back to what they were originally:
From within "Settings" > "Driving" Base Profile >> Navigational Settings >> Map during Navigation >> Auto-centre map view >> 20 seconds
From within "Settings" > "Driving" Base Profile >> Navigational Settings >> Map during Navigation >> Auto Zoom Map >> To LONG RANGE
From within "Settings" > "Driving" Base Profile >> Navigational Settings >> Map during Navigation >> Snap to road >> "ON"

Since putting the settings back to the above settings, I have not had any issue... So I can only assume that there is an ISSUE/BUG when you set the "Auto Zoom Map" setting to "No Auto Zoom"?

Can or has anyone experienced the above issue I described and found the cause to be related to the "No Auto Zoom" Setting?

Not sure if this related to only IOS devices or both IOS & Android devices...

**Comment by DmitryAlexei** [comment]
Your issue is probably an OOM. It can appear, when OsmAnd needs to continuously (re)draw too many tiles on the big tab screen. Please try to disable everything you don't need for navigation in your driving profile (these preferences will affect only driving, other profiles will remain the same):
* you can disable 3D relief, contour lines, hillshades and slopes
* in menu > configure map > details you can disable unneeded details
* in menu > configure map > hide try to mark unneeded objects (boundaries, house numbers, underground objects, POI labels and POI icons, etc.)
* set map magnifier by 100% (menu > configure map > map magnifier) (this may be the main issue)
